config/locales/en.yml

```
@@ -712,6 +712,9 @@ en:
         revenue: Revenue
         source: Source
         title: RailsDevs revenue
+  pay_mailer:
+    subscription_renewing:
+      subject: Your RailsDevs subscription renews in 3 days. Have you made a hire?
```

```suggestion
      subject: Your RailsDevs subscription renews soon. Have you made a hire?
```

Just in case they see it a day later.

---

👍 done